runtime/make/Makefile.runtime.include

```diff
@@ -127,7 +127,11 @@ ifeq ($(CHPL_LIB_PIC),pic)
 RUNTIME_DEFS += $(SHARED_LIB_CFLAGS)
 endif
 
-RUNTIME_CFLAGS += $(RUNTIME_DEFS)
-RUNTIME_CXXFLAGS += $(RUNTIME_DEFS) $(CXXFLAGS)
-
+# If compiling for GPU locale add CUDA runtime headers to include path
+ifeq ($(CHPL_LOCALE_MODEL),gpu)
+RUNTIME_INCLS += -I/usr/local/cuda/include
+RUNTIME_DEFS += -DHAS_GPU_LOCALE
+endif
 
+RUNTIME_CFLAGS += $(RUNTIME_DEFS)
+RUNTIME_CXXFLAGS += $(RUNTIME_DEFS) $(CXXFLAGS)
```

Comment:
It's nice to add newlines at the end of files so github doesn't show us the red symbol. (Not sure it matters for any other reason but AFAIK we tend to have newlines at the end in general).

Reply:
Got it, I'll try and remember this going forward.